CHANGELOG.md

```diff
@@ -1,6 +1,21 @@
 Change Log
 ==========
 
+# Version 3.8.3
+
+_2024-03-20_
+
+Version 3.8.3 is a maintenance release with two new convenience APIs and a Kotlin update. New developments happen in the 4.x versions.
+
+Note: this release upgrades the library's Kotlin dependency to 1.9 - this means projects using Apollo can depend on Kotlin 1.8+.
```

Comment:
Technically, they could depend on 1.8+ before too. The different is that now they have to.

```
Note: because apollo-kotlin depends on kotlin-stdlib version 1.9, you will need the 1.8+ Kotlin compiler to compile your project.
```

Reply:
Woops sorry I merged too fast. I'm about to open another PR for the README, I'll update the sentence there

Reply:
My bad, I commented to fast as well. Thanks for updating!

Reply:
Also just updated my comment to remove future tense 😄